systemtest/src/test/java/io/strimzi/systemtest/upgrade/StrimziUpgradeST.java

```diff
@@ -208,6 +215,20 @@ void testUpgradeKafkaWithoutVersion() throws IOException {
                 .stream().filter(c -> c.getName().equals("kafka")).findFirst().get().getImage(), containsString("2.6.0"));
```

Comment:
Can this be some constant somewhere on the beginning? (the version I mean)

Reply:
Sure